File: Python/1036.Bhaskara'sFormula.py
import sys
import math

a, b, c = map(float, input().split())

if(a == 0):
    print("Impossivel calcular")
    sys.exit()

delta = pow(b, 2) - (4*a*c)

if(delta < 0):
    print("Impossivel calcular")
else:
    r1 = (-b + math.sqrt(delta))/(2*a)
    r2 = (-b - math.sqrt(delta))/(2*a)
    print("R1 = {:.5f}".format(r1))
    print("R2 = {:.5f}".format(r2))

File: C/1036.Bhaskara'sFormula.c
#include <stdio.h>
#include <math.h>

int main(void){
    double a, b, c, delta, r1, r2;

    scanf("%lf %lf %lf", &a, &b, &c);

    if(a == 0){
        printf("Impossivel calcular\n");
    }

    delta = pow(b, 2) + (4*a*c);

    if(delta < 0){
        printf("Impossivel calcular\n");
    }else if(delta > 0){
        r1 = (-b + sqrt(delta))/(2*a);
        r2 = (-b - sqrt(delta))/(2*a);
        printf("R1 = %.5f\nR2 = %.5f\n", r1, r2);
    }

    return 0;
}